scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -33,7 +35,7 @@ odo login -u developer -p developer
 oc whoami
 
 # import the odo-init-image for s390x arch
-if [ "${ARCH}" == "s390x" ]; then
+if [ "${ARCH}" == "s390x" ] || [ "${ARCH}" == "ppc64le" ]; then
     export ODO_BOOTSTRAPPER_IMAGE=registry.redhat.io/ocp-tools-4/odo-init-container-rhel8:1.1.4
```

Comment:
We should soon be able to push this image to unauthenticated registry(registry.access.redhat.com). It should not than more than a day or 2 to happen. So maybe we can hold on that. Meanwhile can you update the registry?
/hold

Reply:
Ok will wait until we push this image to unauthenticated registry(registry.access.redhat.com). Any update needed from my side?

Reply:
Probably the registry change. But I will tell you when

Reply:
@mohammedzee1000 I can see this PR # https://github.com/openshift/odo/pull/3688 merged. Can I go ahead and make the necessary change to use the multi arch image which is now the default image for odo, instead of the `ODO_BOOTSTRAPPER_IMAGE` exported here?

Reply:
@mohammedzee1000 I have removed this `ODO_BOOTSTRAPPER_IMAGE ` for both Z and P after confirming with @zhengxiaomei123 that this works for Z as well. The image will now be used from `defaultBootstrapperImage ` updated in `pkg/devfile/adapters/common/utils.go` after merge of PR # #3688. Please review and approve.

Reply:
@sarveshtamba Thanks for your work.